.github/workflows/runner.yml

```diff
@@ -6,6 +6,51 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  dispatch:
+    name: 🐝 Dispatch
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      android: ${{ steps.filter_macos.outputs.android }}
+      ios: ${{ steps.filter_macos.outputs.ios }}
+      linux: ${{ steps.filter_macos.outputs.linux }}
+      macos: ${{ steps.filter_macos.outputs.macos }}
+      windows: ${{ steps.filter_macos.outputs.windows }}
+      web: ${{ steps.filter_macos.outputs.web }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - uses: AurorNZ/paths-filter@298e00d16954592b06fe237dbf749947c3dd4a98
+      id: filter_macos
+      with:
+        filters: |
+          android:
+            - '**'
+            - '!platform/*/**'
+            - 'platform/android/**'
```

Comment:
Not deeply familiar with glob syntax but does `!platform/*/**` exclude `platform` itself?

Reply:
I wasn't 100% sure, but now I am — A change to `/platform/something.file` will still trigger all CI: https://github.com/Ivorforce/godot/actions/runs/12241909569